rfc6962-bis.xml

```diff
@@ -1458,6 +1463,14 @@ but it is expected there will be a variety.
           </t>
         </section>
       </section>
+      <section title="TLS Feature Extension">
+	<t>
+	  If a certificate in a chain includes the <xref target="tls_transinfo_extension">transparency_info</xref> extension number in a <xref target="RFC7633">TLS Feature Extension</xref>, then CT compliance is required for all certificates which are directly or indirectly issued by that certificate.
+	</t>
+	<t>
+	  Clients MUST treat certificates which fail to comply with this requirement as invalid.
```

Comment:
Should have been clearer: What I suggest is you use the term 'non-compliant' rather than 'invalid'.